docs/source/getting-started/installation.rst

```diff
@@ -107,7 +107,7 @@ To use more than one cloud, combine the pip extras:
 
         .. code-block:: shell
 
-          pip install -U "skypilot[aws,gcp]"
+          pip install -U "skypilot[kubernetes,aws,gcp]"
```

Comment:
add `kubernetes` extras to nightly and from source tabs too?

Reply:
Done.